osutils.pm

```diff
@@ -78,28 +78,26 @@ sub quote {
 }
 
 sub run {
-    bmwqemu::diag "running " . join(' ', @_);
-    my @args = @_;
-    my $out;
-    my $buffer;
-    open my $handle, '>', \$buffer;
-    my $p = process(sub { local *STDERR = $handle; exec(@args) });
-    $p->channels(0)->quirkiness(1)->internal_pipes(0)->separate_err(0)->start;
-    $p->on(stop => sub {
-            while (defined(my $line = $p->getline)) {
-                $out .= $line;
-            }
-            bmwqemu::diag $buffer if defined $buffer && length($buffer) > 0;
-    });
-    $p->wait_stop;
+    my @cmd = @_;
+
+    bmwqemu::diag "Run command `@cmd`";
+    my $p = process(sub { exec(@cmd) });
+    $p->quirkiness(1)->separate_err(0)->start()->wait_stop();
+
+    my $stdout = join('', $p->read_stream->getlines());
```

Comment:
I assume this is to workaround https://github.com/mudler/Mojo-IOLoop-ReadWriteProcess/pull/16.

Reply:
Right